pkg/helm/utils.go

```diff
@@ -58,7 +67,124 @@ func GetHelmReleaseVersion(config *HelmConfig, releaseName string) (string, erro
 	return history[0].Chart.Metadata.Version, nil
 }
 
-func CreateTensorleapChartValues(params *ServerHelmValuesParams) Record {
+//go:embed resources/*
+var dictFiles embed.FS
+
+func readFileToList(fs embed.FS, filePath string) ([]string, error) {
+	fileData, err := fs.ReadFile(filePath)
+	if err != nil {
+		return nil, err
+	}
+	scanner := bufio.NewScanner(strings.NewReader(string(fileData)))
+	var lines []string
+	for scanner.Scan() {
+		lines = append(lines, scanner.Text())
+	}
+	if err := scanner.Err(); err != nil {
+		return nil, err
+	}
+	return lines, nil
+}
+
+func loadWordList(listName string) ([]string, error) {
+	fileName := fmt.Sprintf("resources/%s_en.txt", listName)
+	list, err := readFileToList(dictFiles, fileName)
+	if err != nil {
+		fmt.Println("Error reading file:", err)
+		return nil, err
+	}
+	return list, nil
+}
+
+func loadAdjectiveList() ([]string, error) {
+	return loadWordList("adjectives")
+}
+
+func loadAnimalList() ([]string, error) {
+	return loadWordList("animals")
+}
+
+func generateRandomName(seed *int64) (string, error) {
+	var s int64
+	if seed != nil {
+		s = *seed
+	} else {
+		s = time.Now().UnixNano()
+	}
+	var r = rand.New(rand.NewSource(s))
+	adjectives, err := loadAdjectiveList()
+	if err != nil {
+		fmt.Println("Error generating random name:", err)
+		return "", err
+	}
+	adjective := adjectives[r.Intn(len(adjectives))]
+
+	animals, err := loadAnimalList()
+	if err != nil {
+		fmt.Println("Error generating random name:", err)
+		return "", err
+	}
+	animal := animals[r.Intn(len(animals))]
+	return fmt.Sprintf("%s-%s", adjective, animal), nil
```

Comment:
Why are we generating such complicated names and not just using some randomly generated uuid?

Reply:
@lavie  Consider using the computer username

Reply:
They're less complicated than UUID. Easier to remember, to type, to recall... 
Similar to how docker gives random human-readable container names because typing and remembering a container ID is much harder.